### PR TITLE
feat(router): 增加 xiaote.com 论坛路由

### PR DESCRIPTION
**Pull request by WxSimon**
## 该 PR 相关 Issue / Involved Issue

Close #

## 路由地址示例 / Example for the Proposed Route(s)

```routes
/xiaote/news
```

## 新 RSS 路由检查表 / New RSS Route Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/advanced/script-standard) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/advanced/script-standard)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [ ] 全文获取 full article
  - [ ] 使用缓存 Use cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/advanced/pub-date) [date and time](https://docs.rsshub.app/en/joinus/advanced/pub-date)
  - [x] 可以解析 Parsed
  - [x] 时区正确 Correct time zone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/v2/xiaote/index.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            const item = node.node;
+            let description = item.content;
+            if (item.imageUrls) {
+                item.imageUrls.forEach((url) => {
+                    description += `<img src="${url}">`;
+                });
+            }
+
+            return {
+                title: item.content.substring(0, Math.max(item.content.length, 12)) + '...',
```
Comment (TonyRL) [inline lib/v2/xiaote/index.js:40]: RSSHub will [trim title](https://github.com/DIYgod/RSSHub/blob/04820c33d006c2d1afb88c52b20b0561a4171086/lib/middleware/template.js#L70) if it's longer than `TITLE_LENGTH_LIMIT`.

File: lib/v2/xiaote/maintainer.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    xiaote: ['wxsimon'],
```
Comment (TonyRL) [inline lib/v2/xiaote/maintainer.js:2]:
```suggestion
    '/news': ['wxsimon'],
```

File: lib/v2/xiaote/radar.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'xiaote.com': {
+        _name: '小特社区',
+        '.': [
+            {
+                title: '首页帖子',
+                docs: 'https://docs.rsshub.app/bbs#xiao-te-she-qu',
+                source: ['/'],
+                target: '',
```
Comment (TonyRL) [inline lib/v2/xiaote/radar.js:9]:
```suggestion
                target: '/xiaote/news',
```

File: website/docs/routes/bbs.md [hunk 4]
```diff
@@ -1087,6 +1087,12 @@ import Route from '@site/src/components/Route';
 
 </Route>
 
+## 小特社区 {#xiao-te-she-qu}
+
+### 首页帖子 {#xiao-te-she-qu-shou-ye-tie-zi}
+
+<Route author="wxsimon" example="/xiaote/news" path="/xiaote/news" paramsDesc={['无']}></Route>
```
Comment (TonyRL) [inline website/docs/routes/bbs.md:1094]:
```suggestion
<Route author="wxsimon" example="/xiaote/news" path="/xiaote/news"></Route>
```

**Comment by wxsimon-stu** [comment]
@TonyRL  Thank you very much. I've edited all the comments.

**Pull request merged**